system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_granule_update_schema_test.clj

```diff
@@ -0,0 +1,76 @@
+(ns cmr.system-int-test.ingest.bulk-update.bulk-granule-update-schema-test
+  (:require
+   [cheshire.core :as json]
+   [clojure.string :as string]
+   [clojure.test :refer :all]
+   [cmr.common.util :as util :refer [are3]]
+   [cmr.message-queue.test.queue-broker-side-api :as qb-side-api]
+   [cmr.mock-echo.client.echo-util :as echo-util]
+   [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+   [cmr.system-int-test.system :as sys]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest]))
+
+(use-fixtures :each (ingest/reset-fixture {"provguid1" "PROV1"}))
+
+(def base-request {:name "sample bulk update"
+                   :operation "UPDATE_FIELD"
+                   :update-field "OPeNDAPLink"
+                   :updates
+                   [["ur_1" "s3://aws.foo.com"]
```

Comment:
This `base-request` is a bad request itself. It provided a `s3` link to an OPeNDAPLink update. We should probably add a new validation to flag this error as well. We should use a base request that is valid and show that it passes the validation.